config/jobs/image-pushing/k8s-staging-external-dns.yaml

```diff
@@ -21,4 +21,5 @@ postsubmits:
               - --project=k8s-staging-external-dns
               - --scratch-bucket=gs://k8s-staging-external-dns-gcb
               - --env-passthrough=PULL_BASE_REF
+              - --version-tag-filter=v*
```

Comment:
Don't add this until the new `image-builder` image is being used. It won't be recognized until then so this will fail.